### DISCUSSION TITLE
Dummy launch without pytorch (or general) main execution functionality?

### DISCUSSION THREAD

**Discussion by BrechtCorbeel** [Comfy-Org/ComfyUI#4569] [General]
I am using the or the current version of queue saving / queue saver that you can find here:

https://github.com/pkpkTech/ComfyUI-SaveQueues

now the thing is when I run comfy and add to the queue it starts running immediately.

now basically what I want to do is run comfy and add to the queue without it running so I can create queue lists.

mainly because creating batches that I can then alter after wards can save me a lot of queue generation time.

For example right now I am changing parameters with "replace all" in VSCODE so that I can then load the same queues into comfy with changed parameters that I changed enmasse creating multiple files that then are loaded instead of me doing all that manually.

As of right now I have to do this as the generator is running.

**Comment by ltdrdata** [comment]
This is an interesting idea.

**Reply by BrechtCorbeel** [reply]
> This is an interesting idea.

Maybe something like Queue tabs you can select to store Queues, like right now I am finishing and doing a new batch, but I have to add it to the new queue meaning getting this queue back after saving will be a hassle, among which the instant start issue I have to then figure out where this batch started while the queue is running, freezing and lagging, whenever a ksampler is being triggered.
